### PR TITLE
Docker automatic pruning

### PR DESCRIPTION
**Pull request by Tthomas63**
- [X] Creates a new variable in settings `DOCKER_MAX_SIZE_GB` ( Default is 10.0 GB )
- [X] Checks docker images size, if greater than the above settings, forces a prune.
- [X] Branched off of develop instead of the branch for ensuring legacy competition stability with a submission script.

image

### CODE REVIEW

**Comment by Tthomas63** [comment]
@ckcollab This is ready for review

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 1]
```diff
@@ -151,7 +151,7 @@
            var maxLength = 128;
            var text = $(this).val();
            var textLength = text.length;
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: while we're here, can we change this variable from CamelCase to snake_case?

File: codalab/codalabtools/compute/worker.py [hunk 2]
```diff
@@ -55,6 +57,16 @@ def _find_only_folder_with_metadata(path):
                 return folder
 
 
+def docker_prune():
+    """Runs a prune on docker if our images take up more than what's defined in settings."""
+    docker_image_size = os.popen("docker system df | awk -v x=4 'FNR == 2 {print $x}'").read().strip()
```
Comment (ckcollab) [inline codalab/codalabtools/compute/worker.py]: can get rid of `docker_` prefixing all of these variable names

Comment (ckcollab) [inline codalab/codalabtools/compute/worker.py]: Also, let's pair today and write tests for this

**Pull request merged**